ivy/functional/backends/jax/device.py

```diff
@@ -123,6 +123,19 @@ def tpu_is_available() -> bool:
     return _dev_is_available("tpu")
 
 
+def handle_soft_device_variable(*args, **kwargs):
+    default_device = as_native_dev(ivy.default_device())
```

Comment:
You can pass `as_native=True` to `ivy.default_device` which does the same thing as the above line but just looks a bit cleaner I think.